rest/admin_api.go

```diff
@@ -153,6 +153,39 @@ func (h *handler) handleReplicate() error {
 		return err
 	}
 
+	client := &http.Client{}
```

Comment:
No need to create a new HTTP client - can use `HTTPClient` field on server context (`h.server`) so it can be overridden for tests, etc.